downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.6.3 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-6-3/
+active: 2201-6-3
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-6-3
+    - /downloads/swan-lake-release-notes/2201.6.3/
+    - /downloads/swan-lake-release-notes/2201-6-3-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-6-3-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.6.3
+
+<em>Swan Lake 2201.6.3 is the third patch release of Ballerina 2201.6.0 (Swan Lake Update 6) and it includes a new set of bug fixes to the language,
```

Comment:
```suggestion
<em>Swan Lake 2201.6.3 is the third patch release of Ballerina 2201.6.0 (Swan Lake Update 6) and it includes a new set of bug fixes to the language,
```
Do we have any language fixes in addition to the security improvements? If so, we have to add the `Language updates` section mentioning them.

Reply:
Fixed in [49ddc](https://github.com/ballerina-platform/ballerina-dev-website/pull/8201/commits/49ddc42a053a7c8dbe08aaea55962ecb901c2311).